include/cantera/oneD/IonFlow.h

```diff
@@ -31,7 +31,11 @@ namespace Cantera
 class IonFlow : public StFlow
 {
 public:
-    IonFlow(IdealGasPhase* ph = 0, size_t nsp = 1, size_t points = 1);
+    IonFlow(ThermoPhase* ph = 0, size_t nsp = 1, size_t points = 1);
+
+    //! Delegating constructor
+    IonFlow(shared_ptr<Solution> sol, size_t nsp = 1, size_t points = 1);
```

Comment:
The "delegating" aspect is an implementation detail that I don't think needs to be documented here. But if you wanted to add an actual docstring for this constructor, I wouldn't object.

Reply:
:+1: removed.